packages/pwa-buildpack/lib/WebpackTools/MagentoResolver.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
             modules: [options.paths.root, 'node_modules'],
             mainFiles: ['index'],
             mainFields: ['esnext', 'es2015', 'module', 'browser', 'main'],
-            extensions: ['.wasm', '.mjs', '.js', '.json', '.graphql']
+            extensions: ['.wasm', '.mjs', '.js', '.jsx', '.json', '.graphql']
```

Comment:
Doing this part of https://github.com/magento/pwa-studio/blob/a34adc181b065e83f884f7afbf29a2dd859dd8b9/jest.config.js#L68